config/tracking.tsx

```diff
@@ -0,0 +1,79 @@
+export const GA_TRACKING_CODE = process.env.GA_TRACKING_CODE;
```

Comment:
since this file does not include any react components, we can use the .ts file extension here.

Reply:
Yeah clearly, overlooked that!

Reply:
Fixed this!